src/event-log/event-log-level.ts

```diff
@@ -0,0 +1,80 @@
+import type { ULID } from 'ulid';
+import type { Event, EventLog } from './event-log.js';
+
+import { base32crockford } from '@scure/base';
+import { Encoder } from '../utils/encoder.js';
+import { monotonicFactory } from 'ulid';
+import { createLevelDatabase, LevelWrapper } from '../store/level-wrapper.js';
+
+
+type EventLogLevelConfig = {
+ /**
+   * must be a directory path (relative or absolute) where
+   *  LevelDB will store its files, or in browsers, the name of the
+   * {@link https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase IDBDatabase} to be opened.
+  */
+  location: string,
+  createLevelDatabase?: typeof createLevelDatabase,
+};
+
+export class EventLogLevel implements EventLog {
+  config: EventLogLevelConfig;
+  level: LevelWrapper<string>;
+  ulid: ULID;
+
+  constructor(config: EventLogLevelConfig = { location: 'EVENTLOG' }) {
+    this.config = {
+      createLevelDatabase,
+      ...config,
+    };
+
+    this.level = new LevelWrapper<string>({ ...this.config, valueEncoding: 'utf8' });
+    this.ulid = monotonicFactory();
+  }
+
+  async open(): Promise<void> {
+    // `db.open()` is automatically called by the database constructor.  We're calling it explicitly
```

Comment:
is this needed?  we already have this comment in `IndexLevel`

Reply:
na. was just a note to myself while implementing the event log. removed it. feel free to resolve after taking a look.